website/docs/docs/dbt-versions/release-notes/07-June-2023/product-docs-jun.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "June 2023 product docs updates"
+description: "June 2023: The Product docs team merged 132 PRs, made various updates to dbt Cloud and Core, such as the Deploy sidebar, Supported platforms page, added a landing page on the References section, added an ADO example to the CI/CD guide, and more"
+sidebar_label: "Update: Product docs changes"
+tags: [June-2023, product-docs]
+date: 2023-07-04
+sidebar_position: 10
+---
+
+Hello from the dbt Docs team: @mirnawong1, @matthewshaver, @nghi-ly, and @runleonarun! First, we’d like to thank the 17 new community contributors to docs.getdbt.com!
```

Comment:
curious, should we list the new contributors here (@ mention them)? i think it might be a good shoutout for them ❇️

Reply:
i think that's a wonderful idea @nghi-ly !!!!!! jerco does this as well for the core releases ✨

Reply:
🙌 

didn't know that about core releases -- cool!